diff --git a/l10n_client_ui/l10n_client_ui.module b/l10n_client_ui/l10n_client_ui.module
index af05bb4..5777d12 100644
--- a/l10n_client_ui/l10n_client_ui.module
+++ b/l10n_client_ui/l10n_client_ui.module
@@ -27,7 +27,7 @@ function l10n_client_ui_access_tools() {
  */
 function l10n_client_ui_toolbar() {
   if (!l10n_client_ui_access_tools()) {
-    return FALSE;
+    return;
   }
 
   $tab['l10n_client_ui'] = array(
